crates/matrix-sdk-crypto/src/store/integration_tests.rs

```diff
@@ -433,7 +432,7 @@ macro_rules! cryptostore_integration_tests {
                     assert_eq!(loaded.len(), 2, "Candy shouldn't be part of the store");
 
                     assert!(loaded_alice.dirty, "Alice should be considered to be dirty");
-                    assert!(loaded_alice.dirty, "Bob should not be considered to be dirty");
+                    assert!(loaded_bob.dirty, "Bob should not be considered to be dirty");
```

Comment:
Note: the assert comment disagrees with the code here; who's right?

Reply:
I strongly assume the English here is correct, because a) the `loaded_bob` variable was unused, and b) the test still passes after my change...

Reply:
No wait, I see what you mean! Looking into it.

Reply:
@poljar it looks like you wrote this code originally. Did you mean `assert!(loaded_bob.dirty)` or `assert!(!loaded_bob.dirty)` ? The comment suggests that you mean with the `!` but the test passes without the `!`.

Reply:
@bnjbvr is right, Bob shouldn't be dirty. But the tests pass because this line is also incorrect:

```rust
let loaded_bob = loaded.get(alice).expect("Bob should be in the store as as tracked user");
```

We load `alice` twice.

Not that I remember things, my brain is as rotten as it gets :fish:.

Reply:
Wow, well spotted @poljar .

I've applied the fix here, and made the assertion match its description, and it passes for me locally.